src/backend/looped/self/super_hero_fly.cpp

```diff
@@ -22,7 +22,7 @@ namespace big
 
 		bool m_timer = false;
 		std::chrono::system_clock::time_point m_time_stamp;
-		std::chrono::milliseconds m_duration;
+		std::chrono::duration<double> m_duration;
```

Comment:
std::chrono::milliseconds types as std::chrono::duration<long long>, why are you changing it to [double](https://en.wikipedia.org/wiki/Double-precision_floating-point_format), which is a double precision float?

Reply:
One of these changes fixes the features, just reverted it.